### PR TITLE
More gates aqt

### PR DESCRIPTION
**Pull request by ma5x**
Added the R-gate (AQT's version of Cirq's 'PhasedXPowGate) to the set of physical gates for AQT devices.

Also fixed a bug in AQT's data return format.

### CODE REVIEW

**Comment by vtomole** [comment]
Not related to this pull request: Are there plans for fix these issues? https://github.com/quantumlib/Cirq/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+aqt

File: cirq/aqt/aqt_sampler.py [hunk 1]
```diff
@@ -82,15 +82,20 @@ def _generate_json(
...
         circuit = resolve_parameters(circuit, param_resolver)
         for op in circuit.all_operations():
             line_qubit = cast(Tuple[LineQubit], op.qubits)
             op = cast(ops.GateOperation, op)
             qubit_idx = [obj.x for obj in line_qubit]
             op_str = get_op_string(op)
             gate = cast(ops.EigenGate, op.gate)
-            seq_list.append((op_str, gate.exponent, qubit_idx))
+            if op_str == 'R':
+                seq_list.append((op_str, float(gate.exponent),
```
Comment (vtomole) [inline cirq/aqt/aqt_sampler.py]: Why is `float` called here and not in the `else`?

**Comment by ma5x** [comment]
> Not related to this pull request: Are there plans for fix these issues? https://github.com/quantumlib/Cirq/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+aqt

Thanks for bringing this to my attention! On first sight, some of the issues seem to be resolved already, but I will have a closer look.

File: cirq/aqt/aqt_sampler.py (hunk 1 above)
Comment (ma5x) [reply cirq/aqt/aqt_sampler.py]: This is an inconsistency connected to the failed mypy test. Working on it...

Comment (vtomole) [reply cirq/aqt/aqt_sampler.py]: Did we fix this issue yet?

Comment (ma5x) [reply cirq/aqt/aqt_sampler.py]: i think so: The parameter is now cast to float() in both cases.

**Pull request merged**